### DISCUSSION TITLE
Interoperability Write OpenDDSSharp, read with FastDDS

### DISCUSSION THREAD

**Discussion by AlbertoElDevaluador** [jmmorato/openddsharp#203] [Q&A]
Is there any solution to communicate between fastdds and opendds? When writing from opendds, although I see the RTPS packets in Wireshark I do not receive them in the FastDDS program even if I am subscribed to the same topic and everything is a priori correct. I have seen that in other discussions this error is commented. Is it solved? Do you know how to fix it?

**Comment by jmmorato** [comment]
Did you applied the data representation to your DataWriterQos?

```csharp
var dwQos = new DataWriterQos();
dwQos.Representation.Value = new List<short> { DataRepresentationQosPolicy.XCDR_DATA_REPRESENTATION };
```

**Reply by AlbertoElDevaluador** [reply]
Where should i apply it??

**Reply by jmmorato** [reply]
When you create your DataWriter, there is an overload to provide the QoS to be used

**Comment by jmmorato** [comment]
@AlbertoElDevaluador, did you have time to test the solution proposed?
